ui/react-components/components/CustomPopup/CustomPopup.jsx

```diff
@@ -0,0 +1,34 @@
+import Popup from "reactjs-popup";
+import React from "react";
+import PropTypes from "prop-types";
+import classNames from 'classnames';
+import {customPopup} from './CustomPopup.module.scss';
+
+const CustomPopup = props => {
+
+    const {triggerComponent, popupContent} = props;
+
+    const contentStyle = {
+        width: "470px",
```

Comment:
@sowmika148 Any specific reason for not moving this style to css file

Reply:
Cannot be moved. We can discuss

Reply:
Done